host/cli/ps.go

```diff
@@ -35,6 +36,19 @@ func runPs(args *docopt.Args, client *cluster.Client) error {
 	}
 	if args.Bool["-q"] || args.Bool["--quiet"] {
 		for _, job := range jobs {
+			if format := args.String["<format>"]; format != "" {
+				tmpl, err := template.New("format").Funcs(template.FuncMap{
+					"metadata": func(key string) string { return job.Job.Metadata[key] },
+				}).Parse(format)
```

Comment:
The metadata function is job specific, and the template fails to parse without the function defined. I could make the function accept the job (e.g. `{{ metadata . "flynn-controller.app_name" }}`) but it feels a bit awkward like that.